app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```

Comment:
no need for this helper since we aren't using it. You can delete this whole file.